microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -374,7 +383,7 @@ Here are a list of common error codes and possible reasons.  For a full list, se
 |---|---|---|
 | 400 Bad Request | The server didn't understand the request. | There's no package (zip file) in the body.  Or, `Content-Type` header is missing or its value is incorrect. |
 | 401 Unauthorized | The request page needs an authorization. | The auth token is missing, expired, or not valid. |
-| 404 Not Found | The server can't find the requested page. | The specified `productID` or `operationID` isn't valid, or doesn't belong to the developer who is making the request. |
+| 404 Not Found | The server can't find the requested page. | The specified `productID` or `operationID` isn't valid GUID, not valid, or doesn't belong to the developer who is making the request. |
```

Comment:
```suggestion
| 404 Not Found | The server can't find the requested page. | The specified `productID` or `operationID` doesn't have a  valid GUID, isn't valid, or doesn't belong to the developer who is making the request. |
```

Reply:
@yvsubhash let me know if this is accurate, I can commit the change and approve, then we will merge, thanks.